src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { FC, useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import up from '../../asset/icon/expanded-up.svg';
+import down from '../../asset/icon/expanded-down.svg';
+import { classNames } from '../../utils/classNames';
+export type AccordionType = 'default' | 'checkbox';
+export interface IAccordionProps {
+  /** set customized style */
+  className?: string;
+  /** set disabled Accordion */
+  disabled?: boolean;
+  /** set Accordion expanded */
+  expanded?: boolean;
+  /** set action on title clicked */
+  onChange?: () => void;
```

Comment:
This definition is really not strict. Since you are using TS, you would like to define a more detailed onChange function type